apps/11gist-blog/js/script.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+$(document).ready(function() {
+
+  //var PATH_TO_GISTS_JSON = 'api/gists.json';
+  var PATH_TO_GISTS_JSON = 'https://api.github.com/users/arinaldi/gists';
+
+  //put elements into variables
+  var $posts = $('#posts');
+  var $post = $('#post');
+  var $commentsHeader = $('#comments-header');
+  var $comments = $('#comments');
+
+  //get HTML for Mustache templates
+  var postListTemplate = $('#post-list-template').html();
+  var commentsTotalTemplate = $('#comments-total-template').html();
+  var commentTemplate = $('#comment-template').html();
+
+  //define functions to dynamically populate posts and comments
+  function populatePostList(post) {
+    $posts.append(Mustache.render(postListTemplate, post));
+  }
+  function populateCommentsData(data) {
+    $commentsHeader.append(Mustache.render(commentsTotalTemplate, data));
+  }
+  function populateComments(comment) {
+    $comments.append(Mustache.render(commentTemplate, comment));
+  }
+
+  var postListSettings = {
```

Comment:
(nitpicky, but meh) I'd say it'd make more sense to have the `var` here (why no let?) be holding the success callback instead of the whole settings object. If this is the first thing I read, I'm confused because I don't realize it's in this shape because of the jQuery ajax API